source/CILobbyMenu.cpp

```diff
@@ -3,7 +3,7 @@
 //  CoreImpact
 //
 //  Created by Richard Yoon on 4/5/21.
-//  Copyright � 2021 Game Design Initiative at Cornell. All rights reserved.
+//  Copyright � 2021 Game Design Initiative at Cornell. All rights reserved.
```

Comment:
Why was this changed?

Reply:
Im not sure where thats from. Ill revert it back to the original line.